ProjectStructureAndRoles.md

```diff
@@ -51,6 +51,8 @@ Now they can work on the solution and add commits to the branch. If they are don
 6 Finally one of the Codeowners of the Sub Project shall merge the pull request into the main branch. By that the Codeowner closes the pull request. The Codeowner also shall close the branch and the issue (if not already automatically done).
```

Comment:
A pull request review in certain cases might lead to a PR not being approved. In this case, the code owner shall close the pull request with the appropriate comments and the contribution shall not get merged into the main branch.

Reply:
Is added now